dockernet/config/ica_host.json

```diff
@@ -0,0 +1,24 @@
+{
+    "interchainaccounts": {
+        "host_genesis_state": {
+            "active_channels": [],
+            "interchain_accounts": [],
+            "port": "icahost",
+            "params": {
+                "host_enabled": true,
+                "allow_messages": [
+                    "/cosmos.bank.v1beta1.MsgSend",
+                    "/cosmos.bank.v1beta1.MsgMultiSend",
+                    "/cosmos.staking.v1beta1.MsgDelegate",
+                    "/cosmos.staking.v1beta1.MsgUndelegate",
+                    "/cosmos.staking.v1beta1.MsgBeginRedelegate",
+                    "/cosmos.staking.v1beta1.MsgRedeemTokensforShares",
+                    "/cosmos.staking.v1beta1.MsgTokenizeShares",
+                    "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward",
+                    "/cosmos.distribution.v1beta1.MsgSetWithdrawAddress",
+                    "/ibc.applications.transfer.v1.MsgTransfer"
```

Comment:
nit: add "/cosmos.staking.v1beta1.MsgCancelUnbondingDelegation" (since we're encouraging host chains to add it, and some do)

Reply:
done